client/src/Components/RepoBrowser/AzureBlobBrowser.tsx

```diff
@@ -28,6 +29,38 @@ const AzureBlobBrowser = () => {
 
   const onSubmit = async (event) => {
     event.preventDefault();
+    if (account === '' || container === '' || sasToken === '') {
+      toast('Please fill in all blob fields', {
```

Comment:
Please fill in all blob storage account credential fields.

Reply:
Changed